rollup/polyfills/es5.js

```diff
@@ -0,0 +1,40 @@
+// Sentry ES5 polyfills
```

Comment:
My only concern here is that we _may_ interfere with polyfills that users themselves added 🤔 but I guess if that's the case users need to make sure to add the polyfills before they add sentry (which they should have done already anyhow in order for Sentry not to break).

Reply:
👍 then let's :shipit: